ksp-amd64.config

```diff
@@ -0,0 +1,25 @@
+## https://kernsec.org/wiki/index.php/Kernel_Self_Protection_Project/Recommended_Settings
+
+## Full 64-bit means PAE and NX bit.
+CONFIG_X86_64=y
+
+## Disallow allocating the first 64k of memory.
+CONFIG_DEFAULT_MMAP_MIN_ADDR=65536
+
+# Disable Model-Specific Register writes.
+# CONFIG_X86_MSR is not set
+
+## Randomize position of kernel and memory.
+CONFIG_RANDOMIZE_BASE=y
+CONFIG_RANDOMIZE_MEMORY=y
+
+## Modern libc no longer needs a fixed-position mapping in userspace, remove it as a possible target.
+CONFIG_LEGACY_VSYSCALL_NONE=y
+
+## Enable Kernel Page Table Isolation to remove an entire class of cache timing side-channels.
+CONFIG_PAGE_TABLE_ISOLATION=y
+
+## Remove additional attack surface, unless you really need them.
+# CONFIG_IA32_EMULATION is not set
```

Comment:
Well, I dare say `USE=hardened` killing multilib sounds against the principle of least surprise.

Reply:
thanks for catching, I almost never use multilib, looked normal to me.
I also removed removal of x32 and ldt thing in the same section. ldt breaks wine.